target_gsheet.py

```diff
@@ -119,13 +122,15 @@ def flatten(d, parent_key='', sep='__'):
             items.append((new_key, str(v) if type(v) is list else v))
     return dict(items)
 
-def persist_lines(service, spreadsheet, lines):
+def persist_lines(service, spreadsheet, lines, clear_sheets=False):
```

Comment:
we shouldn't pass clear_sheets but instead we should pass the whole config and in config.json we should have sync setup to replace or append.

Reply:
we may not even need to pass the config object because it's already on the class?

Reply:
Done